modules/apps/document-library/document-library-service/src/main/java/com/liferay/document/library/internal/exportimport/data/handler/DLFileEntryTypeStagedModelDataHandler.java

```diff
@@ -454,12 +428,21 @@ private DLFileEntryType _fetchExistingFileEntryTypeWithParentGroups(
 	private static final Log _log = LogFactoryUtil.getLog(
 		DLFileEntryTypeStagedModelDataHandler.class);
 
+	@Reference
 	private DDMStructureLocalService _ddmStructureLocalService;
+
+	@Reference
 	private DLFileEntryTypeLocalService _dlFileEntryTypeLocalService;
 
 	@Reference
 	private GroupLocalService _groupLocalService;
 
+	@Reference(
+		target = "(&(verify.process.name=com.liferay.document.library.service))"
```

Comment:
@JanisZhang Could you check whether `target = "(verify.process.name=com.liferay.document.library.service)"` works?

Basically, when using `&`, we usually write `(&(A)(B))`, meaning `A && B`; if there's only 1 condition, I don't think `&` is needed.